packages/create-flex-plugin/templates/js/src/components/CustomTaskList/CustomTaskList.Styles.js

```diff
@@ -1,4 +1,4 @@
-import { default as styled } from 'react-emotion';
+import styled from 'react-emotion';
```

Comment:
are we going to have different templates for 2.0 later?

Reply:
I don't know - does that matter here?

Reply:
it doesn't matter for this pr but there are definitely different package versions (emotion, react, etc) that we should be using for a plugin that's using flex ui 2.0

Reply:
Can you create a JIRA ticket? We should then do this for this cycle for sure.

Reply:
https://issues.corp.twilio.com/browse/FLEXY-3601